apps/web/app/books/[id]/_components/BookStatusModal.tsx

```diff
@@ -31,48 +35,64 @@ export const BookStatusModal = ({
   onConfirm,
   initialReadState = 'WANT_TO_READ',
 }: BookStatusModalProps) => {
+  const router = useRouter();
+  const path = usePathname().toString().split('/')[1];
+  const hasSearch = path === 'search';
+
   const [activeStatus, setActiveStatus] = useState<READING_STATUS>(initialReadState);
 
+  const { isOpen: isOrbModalOpen, openModal: openOrbModal, closeModal: closeOrbModal } = useModal();
+
   const handleSave = () => {
     onConfirm(activeStatus);
     closeModal();
+    openOrbModal();
   };
```

Comment:
onClose를 외부에서 주입받아서 실행시키는 것도 괜찮을 것 같아여.

```suggestion
  const handleSave = () => {
    onConfirm(activeStatus);
    closeModal();
    onClose();
  };
```

Reply:
이미 `isOpen`, `closeModal`를 외부에서 받고 있어요! 따라서 모달 상태를 외부에서 두개 받는 것보다는 좋지 않을까 생각했어요~